examples/notebooks/persistent_dataset_speed.ipynb

```diff
@@ -311,7 +307,7 @@
     "\n",
     "cache_epoch_num, cache_total_time, cache_epoch_loss_values, cache_metric_values, cache_epoch_times = \\\n",
     "    train_process(cache_train_ds, cache_val_ds)\n",
-    "print('Total training time of {} epochs with CacheDataset: {:.4f}'.format(cache_epoch_num, cache_total_time))"
+    "print(f\"Total training time of {cache_epoch_num} epochs with CacheDataset: {cache_total_time}\")"
```

Comment:
The previous string only keeps 4 places after the decimal point.
Something like: `f"XXX{data: 4}"`

Thanks.